x-pack/plugin/src/test/resources/rest-api-spec/api/cat.ml.datafeeds.json

```diff
@@ -0,0 +1,89 @@
+{
+  "cat.ml.datafeeds":{
+    "documentation":{
+      "url":"http://www.elastic.co/guide/en/elasticsearch/reference/current/ml-get-datafeed-stats.html"
+    },
+    "stability":"stable",
+    "url":{
+      "paths":[
+        {
+          "path":"/_cat/ml/datafeeds",
+          "methods":[
+            "GET"
+          ]
+        },
+        {
+          "path":"/_cat/ml/datafeeds/{datafeed_id}",
+          "methods":[
+            "GET"
+          ],
+          "parts":{
+            "datafeed_id":{
+              "type":"string",
+              "description":"The ID of the datafeeds stats to fetch"
+            }
+          }
+        }
+      ]
+    },
+    "params":{
+      "allow_no_datafeeds":{
+        "type":"boolean",
+        "required":false,
+        "description":"Whether to ignore if a wildcard expression matches no datafeeds. (This includes `_all` string or when no datafeeds have been specified)"
+      },
+      "bytes":{
```

Comment:
Are there any byte units returned?

Reply:
Don't think so, these options are a catch all as these parameters come by default with _cat APIs.

Reply:
There is also a default `size` parameter https://www.elastic.co/guide/en/elasticsearch/reference/current/cat.html that is only used by [cat thread pool](https://github.com/elastic/elasticsearch/blob/91663116221052b9aa05cc183d80c1e546073371/rest-api-spec/src/main/resources/rest-api-spec/api/cat.thread_pool.json#L35)

My preference would be to remove this but it's not a deal breaker